README.md

```diff
@@ -45,6 +45,14 @@ The location at which the `jenkins-cli.jar` jarfile will be kept. This is used f
 
 Jenkins plugins to be installed automatically during provisioning.
 
+``` 
+jenkins_plugins:
+  - blueocean
+  - { name: "influxdb", version: "1.12.1" }
+```
+
+Plugins list can use name or dictionary with `name` and `version` keys to pin specific version of a plugin.
```

Comment:
Can you just replace the plugins example and docs above (and set this line to say "Defaults to an empty list (`[]`)." at the beginning), so there are not two items showing `jenkins_plugins`?

Also, please match the rest of the file's style using four lines for indent rather than the three backticks.

Reply:
👍